src/routes/safe/components/Apps/hooks/useThirdPartyCookies.ts

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect, useRef, useCallback } from 'react'
+
+const THIRD_PARTY_COOKIES_CHECK_URL = 'https://third-party-cookies-test.vercel.app' //TODO: Change URL
+const SHOW_ALERT_TIMEOUT = 10000
+
+const createIframe = (uri: string, onload: () => void): HTMLIFrameElement => {
```

Comment:
Do we not already have a function that does this for Safe Apps?

Reply:
Do we? ... no idea. Something for creating an iframe dynamically?

Reply:
Yes, I believe so. I may be wrong though

Reply:
404 😂

Reply:
Nevermind then.